README.md

```diff
@@ -1,11 +1,11 @@
-![ivadomed Logo](https://github.com/ivadomed/ivadomed/raw/master/images/ivadomed_logo.png)
-
 Integrated framework for medical image analysis with deep learning.
```

Comment:
i would move that below the image and change the sentence for: "`ivadomed` is an integrated framework for medical image analysis with deep learning."